micrometer-core/src/main/java/io/micrometer/core/instrument/binder/system/DiskSpaceMetrics.java

```diff
@@ -33,28 +35,40 @@
  *
  * @author jmcshane
  * @author Johnny Lim
+ * @author Chris Bono
  * @since 1.8.0
  */
 @NonNullApi
 @NonNullFields
 public class DiskSpaceMetrics implements MeterBinder {
+
     private final Iterable<Tag> tags;
-    private final File path;
-    private final String absolutePath;
+    private final List<File> paths;
 
     public DiskSpaceMetrics(File path) {
         this(path, emptyList());
     }
 
+    public DiskSpaceMetrics(List<File> paths) {
```

Comment:
While mostly intuitive, a JavaDoc would be nice. I know we're missing one on the existing constructors - I can follow-up with adding that separately. In the case of this constructor, the paths should be to different partitions, since there's no point getting the same information about the same partition for different paths on that partition.